sysutils/pfSense-Status_Monitoring/files/usr/local/www/status_monitoring.php

```diff
@@ -123,7 +121,7 @@ function createSlug($string) {
 }
 
 //add a new view and make sure the string isn't empty
-if ($_POST['add-view'] && !empty($view_title) && ($view_title != "default")) {
+if ($_POST['add-view'] && !empty($view_title) && !preg_match("/default/i",$view_title)) {
```

Comment:
This change to use preg_match is not the same as previous string comparison. Any string containing the word `default` will match that regular expression. I believe the proper check here would be `(strtolower($view_title) != "default")`